src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderDescriptionController.cs

```diff
@@ -75,8 +76,12 @@ public async Task<ActionResult> UpdateAsync([FromRoute][Required]string orderId,
                 return BadRequest(new ErrorsModel(isValid.Errors.Select(x => new ErrorModel(x.Id, x.Field))));
             }
 
+            var name = User.FindFirst(x => x.Type == JwtClaimTypes.Name).Value;

```

Comment:
Rather than bringing in the identity package just for the claim type name. Apparently there is the property `User.Identity.Name` to retrieve the name of the user. However by default Identity looks for a particular namespace. I found [this](https://www.jerriepelser.com/blog/overriding-name-claimtype-aspnetcore-oidc/) resource that explains it. What do you think?

@SimonLLawrence Making sure you are aware of this as well.

Reply:
👍